server/kit/kitserver.go

```diff
@@ -87,12 +88,31 @@ func NewServer(svc Service) *Server {
 		errs  *errorreporting.Client
 		propr propagation.HTTPFormat
 	)
-	// if in App Engine, initiate an error reporter and the stackdriver exporters
-	if isGAE() {
-		projectID, serviceID, svcVersion := getGAEInfo()
+
+	projectID := googleProjectID()
+	if projectID != "" {
+		svcName, svcVersion := "default", "v1.0"
```

Comment:
From the docs for error reporting, the [service name and version on optional](https://godoc.org/cloud.google.com/go/errorreporting#Config) so let's just leave them empty instead of adding in these default values.

I like the idea of `SERVICE_NAME` and `SERVICE_VERSION` as a standard env variable, though. Let's make sure to reference them in the README for non-GAE users.

Reply:
make sense